libs/api-client-bear/src/catalogue.ts

```diff
@@ -8,10 +8,21 @@ import { XhrModule } from "./xhr.js";
 import { ExecutionModule } from "./execution.js";
 import { IAdHocItemDescription, IStoredItemDescription, ItemDescription } from "./interfaces.js";
 import {
-    GdcCatalog,
-    GdcDataSetsCsv,
-    GdcDateDataSets,
-    GdcVisualizationObject,
+    CatalogItem,
+    ICatalogGroup,
+    ILoadAvailableCatalogItemsParams,
+    ILoadAvailableCatalogItemsResponse,
+    ILoadCatalogGroupsParams,
+    ILoadCatalogGroupsResponse,
+    ILoadCatalogItemsParams,
+    ILoadCatalogItemsResponse,
+    ILoadDateDataSetsParams,
+    ItemDescription as BearItemDescription,
```

Comment:
Looks like the exactly the same type as ItemDescription, we should use only one of them and remove the later one. (we should likely remove the one in `./interfaces` in `api-client-bear` as it just copies some `api-model-bear` interfaces?)

Reply:
Fixed